recipes/flex/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
```suggestion
project(test_package)
```

Reply:
@ericLemanissier could you remove CMAKE_VERBOSE_MAKEFILE? It will be restricted in the future. Why? It helps a lot when debugging code or checking the compiler flags when building, but for regular propose it only increases the log size and we don't read.